src/nft/components/explore/Explore.css.ts

```diff
@@ -106,3 +106,68 @@ export const carouselIndicator = sprinkles({
   borderRadius: 'round',
   display: 'inline-block',
 })
+
+/* Value Prop Styles */
+export const valuePropWrap = style([
+  {},
+  sprinkles({
+    width: 'full',
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderColor: 'grey700',
```

Comment:
does this border change in light mode vs dark mode?

Reply:
Same in both, but it is an rgba value rather than a simple grey so I added it to the sprinkles.

Reply:
looks like you added it to sprinkles without changing the bordercolor. Also the rgba value you added corresponds to the same value as the hex for medGray in dark mode

Reply:
I fixed the style to use the new `outline` sprinkle color. The opacity in the rgba value will give a different result on light/dark whereas medGray would be the same in each case.